### PR TITLE
Implement Map Endpoint for API

### PR DESCRIPTION
**Pull request by sampritipanda**
* Implemented Functionality for Accessing the Map Endpoint of the API
* Needed to change the base url as the endpoint for the map api is under `/vis/map.png`.

### CODE REVIEW

File: sample.py [hunk 1]
```diff
@@ -38,6 +40,11 @@
 geocode = l.geocode(['New York', 'Singapore'])
 pprint(geocode)
 
+# Loklak Map API
+png_data = l.map(17.582729, 79.118320, 512, 512, 12, 'Hello')
+path = tempfile.mkstemp(suffix=".png")[1]
+open(path, 'wb').write(png_data)
```
Comment (sudheesh001) [inline sample.py]: Creating a file isn't the required thing from this. what's needed is to return the file content, the raw content.

**Comment by sudheesh001** [comment]
@sampritipanda Well done, I like the code changes but I've just left a note about the way in which we're returning this result, since this is an API that others could use, instead of returning the Image path from their local system, wouldn't it be better to send the image content so that the user can use whatever library he/she has to convert the image, some of them use `shutils` the way you've used it, while some use `Image.save()` from the PIL library, we need to be flexible to both.

**Comment by sudheesh001** [comment]
Oh wait, sorry my bad ! I overlooked the https://github.com/loklak/python-loklak-api/pull/10/files#diff-9bb0ed07b972297543115016182fa342R70
Good work @sampritipanda

**Comment by sampritipanda** [comment]
@sudheesh001 Yes, I included the writing to file part in the sample.py to demonstrate how the user could create a png file after getting the results from the API.

**Pull request merged**